### PR TITLE
archinstall: add pt_PT translation

### PR DESCRIPTION
**Pull request by simao-ferreira**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**
v2.3.0

### CODE REVIEW

File: pages.pt_PT/linux/archinstall.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# archinstall
+
+> Instalador com instruções para Arch Linux
```
Comment (navarroaxel) [inline pages.pt_PT/linux/archinstall.md:3]:
```suggestion
> Instalador com instruções para Arch Linux.
```

File: pages.pt_PT/linux/archinstall.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# archinstall
+
+> Instalador com instruções para Arch Linux 
+> Mais informações em: <https://archinstall.readthedocs.io>.
```
Comment (navarroaxel) [inline pages.pt_PT/linux/archinstall.md:4]:
```suggestion
> Mais informações: <https://archinstall.readthedocs.io>.
```

File: pages.pt_PT/linux/archinstall.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+- Iniciar o instalador guiado:
```
Comment (navarroaxel) [inline pages.pt_PT/linux/archinstall.md:6]: Please use imperative mood:
```suggestion
- Inicia o instalador guiado:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.pt_PT/linux/archinstall.md (hunk 3 above)
Comment (simao-ferreira) [reply pages.pt_PT/linux/archinstall.md:6]: I would keep it in the infinitive form as the remaining descriptions.

**Review by navarroaxel — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**